tests/scripts/pulp_deb/test_deb_sync_publish.sh

```diff
@@ -54,6 +54,22 @@ expect_succ pulp deb distribution create --name "${ENTITIES_NAME}_distro" \
 expect_succ curl "$curl_opt" --head \
   --fail "$PULP_BASE_URL/pulp/content/cli_test_deb_distro/dists/default/Release"
 
+# Add test for the gpgkey parameter
+expect_succ pulp deb remote update \
+  --name "${ENTITIES_NAME}_remote" \
+  --gpgkey "wrong_key" 
+
+expect_fail pulp deb repository sync \
+  --name "${ENTITIES_NAME}_repo"
+
+expect_succ pulp deb remote update \
+  --name "${ENTITIES_NAME}_remote" \
+  --gpgkey "$(gpg --armor --export 'pulp-fixture-signing-key')"
+
+expect_succ pulp deb repository sync \
+  --name "${ENTITIES_NAME}_repo"
```

Comment:
Nice idea. But I'm a tiny bit conflicted, because it's testing pulp_deb and not pulp-deb-cli. I'll leave it to @quba42 to decide.

Reply:
I removed this tests for now